common/tcbinfo.c

```diff
@@ -509,6 +509,201 @@ oe_result_t oe_parse_tcb_info_json(
     return result;
 }
 
+OE_INLINE uint32_t ReadUint32(const uint8_t* p)
+{
+    return p[0] | (p[1] << 8) | (p[2] << 16) | (p[3] << 24);
```

Comment:
Shouldn't this be in reverse order?
That is (p[0] << 24) | (p[1] << 16) | (p[2] << 8) | p[3]

Reply:
It would be good to incorporate the rationale for this order.

Couple of suggestions:
1. All the "read" functions in this file read json data. Only these two functions read non-json data. Consider renaming them so as to not have the "read" prefix.

2. The current coding conventions is to prefix all local functions with an _. See _is_space.